### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bstuder99**
Updates billing integration links with suggestions from https://github.com/kubecost/docs/issues/545

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,22 @@ Below are frequently visited Kubecost documentation pages for both the [Commerci
...
 [Cost monitoring best practices](http://blog.kubecost.com/blog/cost-monitoring/)
 
-### Advanced Configuration
+### Third party billing integration guides
 
-[Allocating out of cluster costs (AWS)](https://guide.kubecost.com/hc/en-us/articles/4407596810519)\
-[Allocating out of cluster costs (Azure)](https://guide.kubecost.com/hc/en-us/articles/4407595936023)\
-[Allocating out of cluster costs (GCP)](https://guide.kubecost.com/hc/en-us/articles/4407601816087)\
-[AWS Spot feed integration](/aws-spot-instances.md#spot-data-feed-integration)
+[Allocating out of cluster costs (AWS)](/aws-cloud-integrations.md)\
```
Comment (srpomeroy) [inline README.md:24]: I may have misunderstood; I thought we were changing the wording from `allocating out of cluster costs` to `billing integrations`?
```suggestion
[Amazon Web Services](/aws-cloud-integrations.md)\
```

File: README.md [hunk 2]
```diff
@@ -8,23 +8,22 @@ Below are frequently visited Kubecost documentation pages for both the [Commerci
...
+[Allocating out of cluster costs (Azure)](azure-out-of-cluster.md)\
```
Comment (srpomeroy) [inline README.md:25]: I may have misunderstood; I thought we were changing the wording from `allocating out of cluster costs` to `billing integrations`?
```suggestion
[Microsoft Azure](azure-out-of-cluster.md)\
```

File: README.md [hunk 3]
```diff
@@ -8,23 +8,22 @@ Below are frequently visited Kubecost documentation pages for both the [Commerci
...
+[Allocating out of cluster costs (GCP)](gcp-out-of-cluster.md)\
```
Comment (srpomeroy) [inline README.md:26]: I may have misunderstood; I thought we were changing the wording from `allocating out of cluster costs` to `billing integrations`?
```suggestion
[Google Cloud Platform](gcp-out-of-cluster.md)\
```

File: README.md (hunk 1 above)
Comment (bstuder99) [reply README.md:24]: Whoops, fixed the header on this section but didn't reword the hyperlink text

**Comment by thomasvn** [comment]
@bstuder99 I think this would be a good opportunity to remove all links that reference "guide.kubecost.com" in this doc?

File: README.md [hunk 4]
```diff
@@ -8,23 +8,22 @@ Below are frequently visited Kubecost documentation pages for both the [Commerci
...
 [Cost monitoring best practices](http://blog.kubecost.com/blog/cost-monitoring/)
 
-### Advanced Configuration
+### Third party billing integration guides
 
-[Allocating out of cluster costs (AWS)](https://guide.kubecost.com/hc/en-us/articles/4407596810519)\
-[Allocating out of cluster costs (Azure)](https://guide.kubecost.com/hc/en-us/articles/4407595936023)\
-[Allocating out of cluster costs (GCP)](https://guide.kubecost.com/hc/en-us/articles/4407601816087)\
-[AWS Spot feed integration](/aws-spot-instances.md#spot-data-feed-integration)
+* [AWS](/aws-cloud-integrations.md)\0
```
Comment (thomasvn) [inline README.md:24]: I don't think you need any of the `\` here. And I think the trailing `0` was also an accident?

Comment (bstuder99) [reply README.md]: Fixed

File: README.md [hunk 5]
```diff
@@ -8,23 +8,22 @@ Below are frequently visited Kubecost documentation pages for both the [Commerci
...
-### Quick Installation
+### Quick installation
 
-Check out our [Installation options](https://guide.kubecost.com/hc/en-us/articles/4407601821207) to start monitoring and managing your spend in minutes.
+Check out our [Installation options](/install.md) to start monitoring and managing your spend in minutes.
 
-### Getting Started
+### Getting started
 
 [Troubleshooting installation](https://guide.kubecost.com/hc/en-us/articles/4407601830679)\
```
Comment (thomasvn) [inline README.md:17]: Could we fix these links so that they no longer user "guide.kubecost.com"? Otherwise, looks good to me!

**Comment by bstuder99** [comment]
Ok, links should all be updated. Using direct paths instead of docs.kubecost links in case we move files out of their existing folders so it doesn't break all the links

**Review by srpomeroy — APPROVED** [review]
LGTM!

**Pull request merged**